### DISCUSSION TITLE
[OLD] - [TEMPLATE] Sprint Planning

### DISCUSSION THREAD

**Discussion by Carlson-Lee** [OtagoPolytechnic/forth-street-studios-s1-24-team-1#3] [Sprint Planning]
## Sprint Planning

1. **Introduction and Overview**
   - Welcome and introduction of participants
   - Recap of the sprint goal and objectives

2. **Review of Previous Sprint**
   - Review of completed user stories and tasks
   - Discussion of what went well and areas for improvement

3. **Product Backlog Refinement**
   - Review and prioritize items from the product backlog
   - Discuss any new items or changes in priorities
   - Break down selected backlog items into smaller tasks

4. **Capacity Planning**
   - Assess team capacity for the upcoming sprint
   - Consider team member availability, holidays, and other commitments

5. **Task Estimation**
   - Estimate effort required for each task using story points, time-based estimates, or relative sizing
   - Assign tasks to team members based on their skills and availability

6. **Commitment**
   - Decide which backlog items the team will commit to completing during the sprint
   - Confirm sprint backlog and acceptance criteria for each item

7. **Definition of Done (DoD)**
   - Review and reaffirm the Definition of Done
   - Ensure shared understanding of quality standards and criteria for task completion

8. **Planning**
   - Develop a plan for achieving the sprint goal
   - Identify dependencies, risks, and potential obstacles
   - Set expectations for communication and collaboration during the sprint

9. **Closure**
   - Recap of sprint planning decisions and commitments
   - Confirm date and time for the next sprint planning meeting
   - Any final remarks or questions

**Comment by PBWiseman** [comment]
As this is no longer the sprint planning template we are using I have unpinned and closed this discussion.
